### PR TITLE
chore(eslint): add `no-missing-default-export` rule

### PR DESCRIPTION
**Pull request by fedeci**
Based on #330

### CODE REVIEW

File: dev/hub.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-export { default as Hub } from "./git";
+import git from "./git";
+export default git;
```
Comment (fedeci) [inline dev/hub.ts:2]:
```suggestion
export { default } from './git';
```

File: scripts/eslint-plugin-fig-linter/rules/no-missing-default-export.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        hasExport = true;
+      },
+      'ExportSpecifier[exported.name="default"]'(node) {
+        hasExport = true;
+      },
+      "Program:exit"(node) {
+        if (!hasExport) {
+          context.report({
+            node,
+            message: "File must contain one default export.",
```
Comment (QuiiBz) [inline scripts/eslint-plugin-fig-linter/rules/no-missing-default-export.js:19]: The message can be a bit more expressive:
```suggestion
            message: "File must export default the completion spec object",
```

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks, this rule looks great!

**Review by QuiiBz — APPROVED** [review]
Thanks a lot for this one!

**Pull request merged**